From: Sagi Grimberg <sagi@grimberg.me>
Date: Tue, 21 Jun 2016 18:04:19 +0200
Subject: block: Export blk_poll
References: bnc#1023798,FATE#321463
Patch-Mainline: v4.8-rc1
Git-commit: 9645c1a2336bb92751a04454e7565c09c9a06f3c

The new NVMe over fabrics target will make use of this outside from a
module.

Signed-off-by: Sagi Grimberg <sagi@grimberg.me>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Reviewed-by: Steve Wise <swise@opengridcomputing.com>
Signed-off-by: Jens Axboe <axboe@fb.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 block/blk-core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/block/blk-core.c b/block/blk-core.c
index 5ca486f..4880b66 100644
--- a/block/blk-core.c
+++ b/block/blk-core.c
@@ -3387,6 +3387,7 @@ bool blk_poll(struct request_queue *q, blk_qc_t cookie)
 
 	return false;
 }
+EXPORT_SYMBOL_GPL(blk_poll);
 
 #ifdef CONFIG_PM
 /**
-- 
1.8.5.6

